stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/actions/httpclient/Forward.java

```diff
@@ -88,11 +86,6 @@ protected HTTPCarbonMessage createOutboundRequestMsg(Context context) {
 
         String httpVerb = (String) outboundRequestMsg.getProperty(HttpConstants.HTTP_METHOD);
         outboundRequestMsg.setProperty(HttpConstants.HTTP_METHOD, httpVerb.trim().toUpperCase(Locale.getDefault()));
-
-        ObserverContext observerContext = ObservabilityUtils.getParentContext(context);
-        HttpUtil.injectHeaders(outboundRequestMsg, ObservabilityUtils.getContextProperties(observerContext));
-        observerContext.addTags(HttpUtil.extractTags(outboundRequestMsg));
-
```

Comment:
Need to add HttpUtil.checkAndObserveHttpRequest(context, outboundRequestMsg) here.

Reply:
Now HttpUtil.checkAndObserveHttpRequest(context, outboundRequestMsg) is in AbstractHTTPAction.send(). So all actions are going through this method now including Forward and Execute